ui-particles-angular/tsconfig.json

```diff
@@ -17,7 +17,9 @@
     "module": "es2020",
     "lib": ["es2018", "dom"],
     "paths": {
-      "ui-particles-angular": ["dist/ui-particles-angular/ui-particles-angular", "dist/ui-particles-angular"]
+      // Added by angular-cli
+      "ui-particles-angular": ["dist/ui-particles-angular/ui-particles-angular", "dist/ui-particles-angular"],
```

Comment:
🤔  to test with next PR but i think we need to change `ui-particles-angular` to `@gravitee/ui-particles-angular`